element_calcium_imaging/imaging_no_curation.py

```diff
@@ -304,10 +306,10 @@ def make(self, key):
 
                 params = (ProcessingTask * ProcessingParamSet & key).fetch1("params")
                 sampling_rate = (
-                    ProcessingTask * scan.Scan * scan.ScanInfo & key
+                    ProcessingTask * scan.ScanInfo & key
                 ).fetch1("fps")
 
-                ndepths = (ProcessingTask * scan.Scan * scan.ScanInfo & key).fetch1(
+                ndepths = (ProcessingTask * scan.ScanInfo & key).fetch1(
```

Comment:
Is the `ProcessingTask` needed in this query?
```suggestion
                ndepths = (scan.ScanInfo & key).fetch1(
```

Reply:
Updated, thanks.